### PR TITLE
Make no intersection amongst decks

### PR DESCRIPTION
**Pull request by hellais**
* Create 2 new decks that are currently quite weak, but can be expanded

### CODE REVIEW

**Comment by hellais** [comment]
I did this thing of created two new "filler" decks for IM and tor, so that when we have more complete tests for instant messaging apps and tor we can push this to users that have opted in to testing these categories of apps.

File: data/decks/web.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-name: Web test deck without HTTP Invalid Request Line
-description: This deck runs HTTP Header Field Manipulation, and the Web Connectivity test
+name: Web test deck
+description: This deck runs HTTP Header Field Manipulation and the Web Connectivity test
```
Comment (darkk) [inline data/decks/web.yaml]: By the way, is `${probe_cc}` cached for a long time? I mean, how will `ooni-probe` behave if the laptop is suspended in one country and awaken in another?

Comment (hellais) [inline data/decks/web.yaml]: We cache geoip lookups for up to 10 minutes, so unless a user is travelling very fast between different locations this should not create issues.

**Comment by hellais** [comment]
Got the LGTM. Mergin'.

**Pull request merged**